modules/services/pass-secret-service.nix

```diff
@@ -16,33 +15,48 @@ in {
     package = mkPackageOption pkgs "pass-secret-service" { };
 
     storePath = mkOption {
-      type = with types; nullOr str;
-      default = null;
```

Comment:
Why the change in type?

Reply:
See below comment

Reply:
Reverted in [93d20b1](https://github.com/nix-community/home-manager/pull/3953/commits/93d20b13a018f3fdb5c8e37102b09552c31432b1)